assets/js/theme/gift-certificate.js

```diff
@@ -66,7 +66,7 @@ export default class GiftCertificate extends PageManager {
 
                     cb(numberVal >= min && numberVal <= max);
                 },
-                errorMessage: `You must enter a certificate amount between ${minFormatted} and ${maxFormatted}.`,
+                errorMessage: `${this.context.amountRange} ${minFormatted} ${this.context.and} ${maxFormatted}.`,
```

Comment:
Thank you, This is a much needed PR for translation. 
Just wondering with this {{lang 'common.and'}} , not all languages would use "and" in the same place in different contexts. While it can work here, would there be a way for it to be used very carefully in the future so that it will still remain translate able?

Reply:
Thanks, Chris. You're right. I will try to figure out how to improve this.

Reply:
what I've come up with is adding kind of value placeholders MIN and MAX in en.json (in our case, "You must enter a certificate amount between [MIN] and [MAX]"). The phrase will be modified later on in JS and numbers will be injected instead of placeholders. I hope it will help with translation and we won't depend on word order. The only rule - not to delete MIN and MAX placeholder in json file. 
We used this approach in different place so it will be more or less consistent.